src/vimext.rs

```diff
@@ -1,14 +1,17 @@
 use crate::language_client::LanguageClient;
+use crate::types::LCNamespace;
 use failure::Fallible;
 
 impl LanguageClient {
-    pub fn get_or_create_namespace(&self) -> Fallible<i64> {
-        if let Some(namespace_id) = self.get(|state| state.namespace_id)? {
+    pub fn get_or_create_namespace(&self, ns: &LCNamespace) -> Fallible<i64> {
```

Comment:
Is there some commits missing from here?

I don't see this updated function ever referenced.

Reply:
It gets used by the new `textDocument_semanticHighlight` and `languageClient_handleCursorMoved` in `language_server_protocol.rs`. I think github folded the diff for that file due to it being quite long.

Reply:
Ahh, missed that. Thank you.